src/pages/docs/argo-cd/annotations/helm-annotations.md

```diff
@@ -207,10 +210,11 @@ spec:
         valueFiles:
           - app-files/values.yaml
           - $remote-values/values.yaml
-
+      name: helm-source
     - repoURL: https://github.com/another-repo/values-files-here
       targetRevision: main
       ref: remote-values
+      name: ref-source
```

Comment:
this example won't work as the two values files have a different layout.

Reply:
I'll combine the previous path then that should work

`argo.octopus.com/image-replace-paths.helm-source: "{{ .Values.image.name}}:{{ .Values.image.version}}, {{ .Values.different.structure.here.image }}"`